src/scripts/main.js

```diff
@@ -1,11 +1,22 @@
 'use strict';
 
 function waitFor(element, eventName) {
-// write your code here
-}
+  return new Promise((resolve) => {
+    element.addEventListener(eventName, (eventClick) => {
+      if (eventClick.target === element) {
+        resolve(`Это было ${eventName}`
+        + ` в элементе: ${element.nodeName}, id: ${element.id}.`);
```

Comment:
Why did you translate it?)
Let's leave it in English to pass the tests
`It was ${eventName} on the element: ${element.nodeName}, id: ${element.id}.`